go.mod

```diff
@@ -78,6 +78,8 @@ require (
 	google.golang.org/grpc v1.42.0
 )
 
+require github.com/armon/circbuf v0.0.0-20150827004946-bbbad097214e
```

Comment:
This should be placed in the main `require()` block?

Reply:
Do you know if we are doing anything wrong so that this isn't added to the correct place automatically?